.azure/README.md

```diff
@@ -44,3 +44,27 @@ done
 ```bash
 ps aux | grep start.sh
 ```
+
+# Machine maintenance
+
+As most of our jobs/checks are running in a docker container the OS/machine get polluted soon d fails to run anymore with error such as
```

Comment:
```suggestion
Since most of our jobs/checks are running in a Docker container, the OS/machine can become polluted and fail to run with errors such as:
```